Sources/FluentKit/Properties/Timestamp.swift

```diff
@@ -9,13 +20,14 @@ public enum TimestampTrigger {
 }
 
 @propertyWrapper
-public final class TimestampProperty<Model>
-    where Model: FluentKit.Fields
+public final class TimestampProperty<Model, Formatter>
```

Comment:
Why is it generic over `Model` where model is of type `Fields`. Shouldn't that be renamed? Kind of confusing :)

Reply:
Pass that query to @tanner0101. That's what he did with all the other property wrappers, so I simply followed suit.

Reply:
Yeah, figured as much. CC @tanner0101 might be good to check in a separate PR, non blocking for this one :)